docs/Introduction/getting-started/other-tutorials.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: nodes.liquid
-section: smartContract
+section: gettingStarted
 date: Last Modified
 title: "Learning Resources"
 permalink: "docs/other-tutorials/"
 ---
```

Comment:
It looks like "Smart Contract Learning Materials" is an empty header? Is that a design problem?

Reply:
Removed and updated the sub-headers.